README.md

```diff
@@ -32,7 +32,76 @@ For good first issues regarding contributing, see [issues labeled with open sour
 
 # Local development environment
 
-Please contribute by writing a document about local development environment. See above.
+This section is still under construction, if you have troubles setting the environment, please contact us (see below section).
+
+## Prerequisites
+
+Project requires Node.js >=12 <13 to be installed; all other dependencies come from NPM via package.json.
+
+You will need Node and npm (npm comes with Node). We recommend [using a node version manager, nvm](https://github.com/nvm-sh/nvm) to install Node, as it allows automatically switching between node versions across projects.
+
+We recommend also installing [direnv](https://direnv.net/), which helps automatically set environment variables as you switch directories.
+
+## Development environment setup
+
+Clone the project from GitHub (make sure to clone it from the directory you wish to contain your project)
+
+```shell
+git clone git@github.com:futurice/symptomradar.git
+```
+
+Install NPM dependencies
+
+```shell
+cd symptomradar
+rm -rf node_modules
```

Comment:
Is this needed here? If you just cloned it and are doing `npm i` for the first time, there shouldn't be any node_modules yet? 🤔

Reply:
That's true! I just assume it can't hurt, if there's no folder nothing gets removed anyway 😂 I'm completely fine removing it!

Reply:
Maybe in that case it's enough that it's also mentioned below in the "If you have some issues with the packages installation" section? Yeah it doesn't hurt to do it but also it's not required, but I don't know, your call 😅

Reply:
Yeah, I wouldn't ask to `rm -rf node_modules` by default. :smile: